File: 프로그래머스/lv1/12918. 문자열 다루기 기본/문자열 다루기 기본.js
function check_string(s)
{
    for (let i = 0 ; i < s.length; i++)
        {
            if((s[i] >='0') && (s[i] <='9'))
                {
                    continue;
                }
            else return false;
        }
    return true;
}
function check_length(s)
{
    if ((s.length == 4) | (s.length == 6))
        {
            return true;
        }
    return false;
}
function solution(s) {
    var answer = true;
    if(check_length(s))
        {
            if(check_string(s))
                {
                    return true;
                }
        }
    return false;

}

File: 프로그래머스/lv1/12918. 문자열 다루기 기본/문자열 다루기 기본.cpp
#include <string>
#include <vector>

using namespace std;

bool solution(string s) {
    bool answer = true;
    if (s.size()== 4 | s.size()==6){
    for ( int i = 0; i < s.size(); i++){
        if (s[i]>=97 &s[i]<=122 | s[i]>=65 &s[i]<=90 ){
            return false;
        }
    }
    }else return false;
    return answer;
}